filebeat/_meta/fields.common.yml

```diff
@@ -3,12 +3,12 @@
   description: >
     Contains log file lines.
   fields:
-    - name: source
-      type: keyword
-      required: true
-      description: >
-        The file from which the line was read. This field contains the absolute path to the file.
-        For example: `/var/log/system.log`.
+    #- name: file.path
```

Comment:
@webmat This small trick you need to do on the Beats side to run the test for the filebeat modules. See also code change below in the log harvester.